src/transformers/generation/utils.py

```diff
@@ -1807,6 +1811,7 @@ def contrastive_search(
         return_dict_in_generate: Optional[bool] = None,
         synced_gpus: bool = False,
         streamer: Optional["BaseStreamer"] = None,
+        low_memory: Optional[bool] = True,
```

Comment:
```suggestion
        low_memory: Optional[bool] = False,
```

(to keep the same behavior by default)

Reply:
Nice catch!